docs/spark/spark-ddl.md

```diff
@@ -98,6 +98,13 @@ CREATE TABLE prod.db.sample
 USING iceberg
 AS SELECT ...
 ```
+```sql
+CREATE TABLE prod.db.sample
+USING iceberg
+PARTITIONED BY (part)
+TBLPROPERTIES ('key'='value')
+AS SELECT ...
```

Comment:
I think it would make more sense to either:
1) Update the existing CTAS statement to use partitioning plus tblproperties.
2) Add a comment in between the two blocks indicating what's different between the two.

I would prefer (1) given that the CTAS syntax is not iceberg specific, this is just covering more Spark DML.

Reply:
hi @kbendick, thx for the advice. I made some changes base on ur suggestion, here is what it looks like now
cc @samredai 

<img width="898" alt="image" src="https://user-images.githubusercontent.com/10529123/164581717-e49c5222-8dd9-4cc8-942e-4b24c5f42fff.png">